src/SamplesApp/UITests.Shared/Windows_UI_Xaml/UIElementTests/UIElement_InitializationSequence_TestControl.cs

```diff
@@ -0,0 +1,65 @@
+﻿using System;
+using Windows.Foundation;
+using Windows.UI.Xaml.Controls;
+
+namespace UITests.Windows_UI_Xaml.UIElementTests
+{
+	public sealed partial class UIElement_InitializationSequence_TestControl : ContentControl
+	{
+		private readonly Func<string, IDisposable> _logger;
+
+		public UIElement_InitializationSequence_TestControl(Func<string, IDisposable> logger, string name)
+		{
+			_logger = logger;
+			Name = name;
+			using (LogRecursively("ctor"))
+			{
+				DefaultStyleKey = typeof(UIElement_InitializationSequence_TestControl);
+
+				Loaded += (snd, evt) => Log("Loaded");
+				Loading += (snd, evt) => Log("Loading");
+				//LayoutUpdated += (snd, evt) => Log("LayoutUpdated");
```

Comment:
Codacy found an issue: [Remove this commented out code.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5850270)